rpc/block_test.go

```diff
@@ -20,6 +20,255 @@ import (
 	"go.uber.org/mock/gomock"
 )
 
+func TestBlockWithTxsAndReceipts(t *testing.T) {
```

Comment:
1. Test cannot be run - compilation error
2. Why you included skipped tests? There is no info with reason for skipping

Reply:
1. Apparently there was an extra argument `n` in my code when I called `rpc.New`
2. The reason is I looked at [TestBlockWithReceipts](https://github.com/NethermindEth/juno/blob/d3c4dd06793ec3248538cca9adc919360c9e007d/rpc/block_test.go#L557) and it had two skipped tests. Honestly, not sure why